main/index.ts

```diff
@@ -313,6 +326,11 @@ app.on('ready', () => {
   menu()
   windows.init()
   if (app.dock) app.dock.hide()
+  if (isDev) {
+    (async () => {
```

Comment:
dont need any of this async/await stuff, can just call this function directly

Reply:
will try

Reply:
runs into dangling promises linting rule but we can use `void`

Reply:
can also just catch the error and print to console